src/components/container/HealthConnectPhrSync/HealthConnectPhrSync.tsx

```diff
@@ -79,7 +81,7 @@ export default function HealthConnectPhrSync(props: HealthConnectPhrSyncProps) {
         return <Button onClick={() => MyDataHelps.showHealthConnectPhrPrompt()}>{language("connect")}</Button>;
     }
 
-    return <Action innerRef={props.innerRef} titleIcon={<img width={15} style={{ marginRight: "8px" }} src={HealthConnectLogo} />}
+    return <Action innerRef={props.innerRef} titleIcon={<img width={15} style={{ marginRight: "8px" }} src={layoutContext.colorScheme == "dark" ? HealthConnectLogoWhite : HealthConnectLogo} />}
```

Comment:
I would create a HealthConnectIcon component and use that in various places.
```suggestion
function HealthConnectIcon({ width, style }) {
    const layoutContext = useContext<LayoutContext>(LayoutContext);
    return <img width={width} style={style} 
        src={layoutContext?.colorScheme == "dark" ? HealthConnectLogoWhite : HealthConnectLogo} 
    />;
}

return <Action innerRef={props.innerRef} titleIcon={<HealthConnectIcon width={15} style={{ marginRight: "8px" }} />}  />
```

Reply:
Yeah this makes sense.  Normally i'd follow rule of 3s and hold off but we will inevitably forget about this the next time we have to do this otherwise